crypto/property/property.c

```diff
@@ -130,9 +131,17 @@ OSSL_PROPERTY_LIST **ossl_ctx_global_properties(OSSL_LIB_CTX *libctx,
                                                 ossl_unused int loadconfig)
 {
     OSSL_GLOBAL_PROPERTIES *globp;
+    OPENSSL_INIT_SETTINGS settings;
+
+    memset(&settings, 0, sizeof(settings));
+    /*
+     * In diagnostic mode, keep going and report success, even if provider
+     * activation fails.
+     */
```

Comment:
I am confused. You set this CONF_MFLAGS_SILENT_ACTIVATION flag by default here but in the documentation you say it is not set by default?